groovy/postInit/components/Batteries.groovy

```diff
@@ -223,54 +156,422 @@ mods.gregtech.assembler.recipeBuilder()
         .EUt(16)
         .buildAndRegister()
 
-mods.gregtech.assembler.recipeBuilder()
-        .inputs(metaitem('battery.hull.lv'))
-        .inputs(ore('dustCadmium'))
-        .inputs(ore('dustNickelOxideHydroxide'))
-        .fluidInputs(fluid('potassium_hydroxide_solution') * 100)
-        .outputs(metaitem('battery.re.lv.cadmium'))
+/*
+ * Components
+ */
+
+// Pb anode & PbO2 cathode line
+
+crafting.addShapeless("anode_lead", metaitem('anode.lead'), [
+        metaitem('plateLead'), metaitem('cableGtSingleTin')
+]);
+
+MIXER.recipeBuilder()
+        .inputs(ore('dustLead'))
+        .fluidInputs(fluid('diluted_sulfuric_acid') * 1000)
+        .outputs(metaitem('cathode.lead_paste'))
+        .fluidOutputs(fluid('water') * 1000)
         .duration(100)
         .EUt(16)
         .buildAndRegister()
 
-mods.gregtech.assembler.recipeBuilder()
-        .inputs(metaitem('battery.hull.mv'))
-        .inputs(ore('dustCadmium') * 2)
-        .inputs(ore('dustNickelOxideHydroxide') * 2)
-        .fluidInputs(fluid('potassium_hydroxide_solution') * 200)
-        .outputs(metaitem('battery.re.mv.cadmium'))
+crafting.addShapeless("cathode_lead", metaitem('cathode.lead'), [
+        metaitem('anode.lead'), metaitem('cathode.lead_paste')
+]);
+
+ELECTROLYTIC_CELL.recipeBuilder()
+        .inputs(ore('plateLead'))
+        .fluidInputs(fluid('water') * 2000)
+        .notConsumable(metaitem('plateCopper'))
+        .notConsumable(fluid('sulfuric_acid') * 1000)
+        .outputs(metaitem('plateOxideCoatedLead'))
+        .fluidOutputs(fluid('hydrogen') * 2000)
+        .duration(80)
+        .EUt(30)
+        .buildAndRegister()
+
+// Ni(OH)2 & NiO(OH) cathode line
+
+BENDER.recipeBuilder()
+        .inputs(ore('foilNickel'))
+        .inputs(ore('dustSmallNickelHydroxide') * 5)
+        .outputs(metaitem('cathode.nioh2'))
         .duration(200)
-        .EUt(16)
+        .EUt(30)
```

Comment:
why 30EU/t tho?

Reply:
IDK how much energy should roll-bonding consume, so I suppose not too much

Reply:
keep it 16EU/t for metal bender recipes then